components/ForYouCard.tsx

```diff
@@ -0,0 +1,117 @@
+  },
+  {
+    title: "Whispers of Ancient Ruins",
+    description: "Exploring the remnants of ancient ruins, I felt the weight of history upon my shoulders. Stone pillars stood tall against the backdrop of a setting sun, their weathered surfaces bearing the marks of time. As I walked among the ruins, I could almost hear the whispers of the past, tales of civilizations long gone. Nature had begun to reclaim its territory, with vines weaving through cracks in the stone. It was a humbling experience, standing amidst the remnants of a bygone era, feeling a deep connection to those who walked these paths centuries before.",
+  },
+  {
+    title: "A Serenade of Birdsong",
+    description: "Waking to a symphony of bird songs, I knew it would be a day filled with nature's melodies. The air was alive with chirps, trills, and calls, each bird adding its unique note to the chorus. I followed the sound through the forest, feeling as though I was part of this avian orchestra. Perched on a moss-covered rock, I closed my eyes and let the music wash over me. It was a reminder of the beauty and harmony of the natural world, a serenade that filled my heart with joy.",
+  },
+  // Add more journal stories as needed
+];
+
+let lastDisplayedIndex = -1;
+
+const getRandomStory = () => {
+  let randomIndex = Math.floor(Math.random() * journalStories.length);
+
+  // If lastDisplayedIndex exists, ensure the next random story is different
+  if (lastDisplayedIndex !== -1) {
+    randomIndex = (lastDisplayedIndex + 1) % journalStories.length;
+  }
+
+  // Update lastDisplayedIndex for next time
+  lastDisplayedIndex = randomIndex;
+
+  return journalStories[randomIndex];
+};
+
+
+
+
+interface ForYouCardProps {}
+
+const ForYouCard: React.FC<ForYouCardProps> = () => {
+  const { title, description } = getRandomStory();
+
+  return (
+    <View style={styles.card}>
+      <Text style={styles.title}>{title}</Text>
+      <Text style={styles.description}>{description}</Text>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  card: {
+    flexGrow: 1,
+    width: '100%',
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#fff',
+    borderRadius: 10,
+    padding: 20,
+    shadowColor: '#000',
+    shadowOffset: {
+      width: 0,
+      height: 2,
+    },
+    shadowOpacity: 0.25,
+    shadowRadius: 3.84,
+    elevation: 5,
+  },
+  title: {
+    fontSize: 24,
+    fontWeight: 'bold',
+    marginBottom: 10,
+  },
+  description: {
+    fontSize: 16,
+    textAlign: 'center',
+  },
+});
```

Comment:
I feel like styling should be in a separate area, but I'm not 100% sure. @colefoster @Justin200104 thoughts?

Reply:
I think globalized styling and component design are both good ideas, will have to be sprint 3 (or 2.5) though unfortunately